trin-storage/src/sql.rs

```diff
@@ -33,7 +34,7 @@ pub const XOR_FIND_FARTHEST_QUERY_NETWORK: &str = "SELECT
                                     content_id_long
                                     FROM content_data
                                     WHERE network = (?2)
-                                    ORDER BY ((?1 | content_id_short) - (?1 & content_id_short)) DESC";
+                                    ORDER BY ((?1 | content_id_short) - (?1 & content_id_short)) DESC LIMIT 1";
```

Comment:
Just to confirm my understanding, but this change doesn't really do much to help with performance, correct? The query takes just as long as it did before, the only difference is that we're returning the first list entry from the query rather than every list entry?

Reply:
Pretty much this should be negligible in most cases but it is best practice to if that is what your returns